Python/Product/EnvironmentsList/images/images.xaml

```diff
@@ -203,4 +203,15 @@
         </Setter>
         <Setter Property="VerticalAlignment" Value="Center" />
     </Style>
+
+    <Style x:Key="DeleteFolderImage" TargetType="Control">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate>
+                    <imaging:CrispImage Moniker="{x:Static catalog:KnownMonikers.DeleteFolder}" />
```

Comment:
Should we get a "delete environment" icon created? I kind of like the implication that an actual folder will be deleted though.

Reply:
I figured the folder one was pretty good, where as a custom delete environment may still be unclear that it deletes it and not just removes it from the list.